generators/server/templates/dotnetcore/src/Project.Infrastructure/Data/Repositories/UnitOfWork.cs.ejs

```diff
@@ -65,8 +67,13 @@ namespace <%= namespace %>.Infrastructure.Data.Repositories
         public void AddGraph<TEntiy>(TEntiy entity) where TEntiy : class
         {
             _context.ChangeTracker.TrackGraph(entity, e => {
+                var alreadyTrackedEntity = _context.ChangeTracker.Entries().FirstOrDefault(entry => entry.Entity.Equals(e.Entry.Entity));
```

Comment:
@ivanmonteiro I have modified this because if two entity have the same relationship an error was throw because the relationship is already track

Reply:
Same